BP/common/landed_titles/BP_landed_titles.txt

```diff
@@ -5001,3 +5001,3668 @@ e_italia = {
 		}
 	}
 }
+
+e_britannia = {
```

Comment:
@LT-Rascek

Reply:
@AvalonXD: Seems to be somewhat close to the proposed layouts of [Roman Britain](https://en.wikipedia.org/wiki/Roman_Britain#/media/File:Roman_Britain_-_AD_400.png), albeit using kingdoms instead of duchies. If you compare e_britannia and the map I posted, there's a number of similarities, with two fewer title (valentia and falvia cesariensis).